javascript/builtins/JSON.json

```
@@ -138,6 +178,87 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "reviver_context_parameter": {
```

So `parse()` can optionally take a reviver function argument. That argument can now take the context parameter. Not sure if this is the right way to record that?

---

This key is a little more self-explanatory:

```suggestion
          "reviver_parameter_context_argument": {
```